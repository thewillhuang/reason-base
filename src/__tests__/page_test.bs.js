// Generated by BUCKLESCRIPT VERSION 2.1.0, PLEASE EDIT WITH CARE
'use strict';

var Jest                 = require("@glennsl/bs-jest/src/jest.js");
var ReasonReact          = require("reason-react/src/ReasonReact.js");
var Page$ReactTemplate   = require("../page.bs.js");
var ReactShallowRenderer = require("bs-react-test-renderer/src/ReactShallowRenderer.bs.js");

describe("App", (function () {
        return Jest.test("renders", (function () {
                      var tree = ReactShallowRenderer.renderWithRenderer(ReasonReact.element(/* None */0, /* None */0, Page$ReactTemplate.make("Hello world!", "will and kate!!!", /* array */[])));
                      return Jest.Expect[/* toMatchSnapshot */16](Jest.Expect[/* expect */0](tree));
                    }));
      }));

/*  Not a pure module */
