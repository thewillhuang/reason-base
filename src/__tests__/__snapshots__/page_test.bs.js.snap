// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders 1`] = `
<div
  className="background text"
  onClick={[Function]}
>
  Hello world! will and kate!!!
</div>
`;
